ntc_templates/templates/alcatel_aos_show_interfaces_port.textfsm

```diff
@@ -1,16 +1,18 @@
 Value PORT ((\d+\/?)+)
-Value WTR_RUNNING (\#?)
 Value PERMANENT_SHUTDOWN (\*?)
 Value ADMIN_STATUS (\S+)
 Value LINK_STATUS (\S+)
 Value VIOLATIONS (\S+)
 Value RECOVERY_TIME (\d+)
 Value RECOVERY_MAX (\d+)
+Value WTR_RUNNING (\#?)
 Value WTR_TIME (\d+)
 Value ALIAS (.*)
 
 Start
-  ^\s*-+\+
-  ^\s*${WTR_RUNNING}${PERMANENT_SHUTDOWN}${PORT}\s+${ADMIN_STATUS}\s+${LINK_STATUS}\s+${VIOLATIONS}\s+${RECOVERY_TIME}\s+${RECOVERY_MAX}\s+${WTR_TIME}\s+"${ALIAS}"\s*$$ -> Record
+  ^\s*-+\+ -> Table
+
```

Comment:
@jmcgill298
Should the Error rule/directive be added to the `Start` state for good measure?

```suggestion
  ^.*$$ -> Error

```